src/main/webapp/js/grapheditor/Format.js

```diff
@@ -4589,7 +4589,7 @@ StyleFormatPanel.prototype.init = function()
 		}
 	
 		this.container.appendChild(this.addStroke(this.createPanel('StyleFormatPanel-addStroke')));
-		this.container.appendChild(this.addLineJumps(this.createPanel('StyleFormatPanel-addLineJumps')));
+		// this.container.appendChild(this.addLineJumps(this.createPanel('StyleFormatPanel-addLineJumps')));
```

Comment:
```suggestion
```

Reply:
It's commented because this line adding this : (it's not supported for the moment)
![image](https://user-images.githubusercontent.com/12846806/146027155-604d8d4d-9ffa-47c5-b73c-dc5d22fee7c5.png)